pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2736,6 +2737,12 @@ self: super: {
       sha256 = "sha256-ZdCXMhni+RGisRODiElObW5c4hKy2giWQmWnatqeRJo=";
     }))
   ];
+
+  # jailbreak to allow deepseq >= 1.5, https://github.com/jumper149/blucontrol/issues/3
+  blucontrol = doJailbreak super.blucontrol;
+
+  # jailbreak to allow rerefined >=0.7.0, https://github.com/raehik/binrep/issues/12
+  binrep = doJailbreak super.binrep;
```

Comment:
should be good to remove next time Hackage pin is updated (binrep-1.1.0 uploaded 2025-03-11T12:11:30Z)

Reply:
Thanks, great, I'll drop the jailbreak from this PR 👍